Formula/sleuthkit.rb

```diff
@@ -11,37 +11,26 @@ class Sleuthkit < Formula
     sha256 "994034b47b0992fa5c651787de90a548edf5a2ccddee7eb4e7a96bd74b23b470" => :el_capitan
   end
 
-  option "with-jni", "Build Sleuthkit with JNI bindings"
-  option "with-debug", "Build debug version"
-
-  depends_on "afflib" => :optional
-  depends_on "libewf" => :optional
-
-  if build.with? "jni"
-    depends_on :java
-    depends_on "ant" => :build
-  end
+  depends_on "ant" => :build
+  depends_on "afflib"
+  depends_on :java
+  depends_on "libewf"
+  depends_on "libpq"
+  depends_on "sqlite"
 
   conflicts_with "irods", :because => "both install `ils`"
   conflicts_with "ffind",
     :because => "both install a 'ffind' executable."
 
   def install
-    ENV.append_to_cflags "-DNDEBUG" if build.without? "debug"
```

Comment:
Do we want to keep this line? We _usually_ don't build stuff with debug symbols/whatnot available by default.

Reply:
Agreed. Restored in https://github.com/Homebrew/homebrew-core/pull/30986/commits/b9168e22ccfb99f02508e9930c09ae7c503ca4e0.